.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ jobs:
           name: Production Deploy
           command: |
             if [ "${CIRCLE_BRANCH}" == "develop" ]; then
-            ssh -o "StrictHostKeyChecking no" deploy@95.85.4.133 'cd feathers-giveth-develop/ && git checkout develop && git pull && [[ -s $HOME/.nvm/nvm.sh ]] && . $HOME/.nvm/nvm.sh && nvm use 10 && npm ci && npm run serve;'
+            ssh -o "StrictHostKeyChecking no" deploy@95.85.4.133 'cd feathers-giveth-develop/ && git checkout develop && git pull && [[ -s $HOME/.nvm/nvm.sh ]] && . $HOME/.nvm/nvm.sh && nvm use 10 && npm ci && ./node_modules/.bin/migrate-mongo up && npm run serve;'
```

Comment:
@aminlatifi 

I've written a migration script for filling `tokenSymbol` for all donations and milestones records, so I think it must be not bad to use `migrate-mongo` tools, and run migrations after every deploy.